### DISCUSSION TITLE
update a record without rewriting everything

### DISCUSSION THREAD

**Discussion by thal-sh** [pocketbase/pocketbase#3863] [Q&A] [answered]
`export async function getCookie(item: any) {
  const existingCookies = cookies().get("cartId")?.value;

    const data = {
      product_id: [
        item,
      ]
    }
    const record = await pb.collection("cartdummy").update(existingCookies, data);
  
}`

this works fine but it overwrites what the whole product_id (which is a foreign key 'many' to products) i want it to add the item on top of what is existing, how? I could technically fetch the cart alone and shove the data like this

`const data = {
      product_id: [
        ...old,
        item,
      ]
    }`
    but im just seeing if there is a simpler way

**Comment by ganigeorgiev** [comment, marked as answer]
You can use the `+` or `-` field modifiers respectively to append or remove rel items:

```js
await pb.collection("cartdummy").update(existingCookies, {
    // or assign an array if you want to add more than 1 id
    'product_id+': item
});
```
